README.md

```diff
@@ -69,9 +76,36 @@ report. Keep this in mind when setting the user roles which can access each repo
   * The API key has to be sent as the value of the `api_key` field in a HTTP POST request.
   * Administrators have the option (in the module system settings) to prohibit non-administrators
     from saving reports where API access is enabled.
+* **Allow public access** \* determines whether the report can be accessed publicly (without a
+  REDCap login)
+  * If the report can be downloaded or retrieved as image, this can be done by all users who access
+    the report at the public URL.
+  * Administrators have the option (in the module system settings) to prohibit non-administrators
+    from saving reports where public access is enabled.
```

Comment:
I think one should add a warning that public reports should not rely on IDs in query strings to filter out rows one does not want to be publicly accessible, as these potentially could be manipulated by the user.

Reply:
There are warnings to this effect in the sections which describe query string smart variables and placeholders.

Reply:
I have added an extra note to the documentation about the security implications of public reports.